runtime/common/src/assigned_slots/mod.rs

```diff
@@ -724,6 +766,9 @@ mod tests {
 			Slots::on_initialize(block);
 			AssignedSlots::on_initialize(block);
 		}
+		//Set the testing MaxTemporarySlots and MaxPermanentSlots values
+		AssignedSlots::set_max_temporary_slots(RuntimeOrigin::root(), 6).unwrap();
+		AssignedSlots::set_max_permanent_slots(RuntimeOrigin::root(), 2).unwrap();
```

Comment:
Normally these are added in a `GenesisConfig`. Since you are adding new storage items, i think it would be appropriate to add that, otherwise the pallet cannot be properly included for a genesis creation.